neo/Prompt/Commands/Wallet.py

```diff
@@ -305,6 +306,66 @@ def command_desc(self):
         return CommandDesc('unspent', 'show unspent assets', params=[p1, p2, p3, p4])
 
 
+class CommandWalletSplit(CommandBase):
+
+    def __init__(self):
+        super().__init__()
+
+    def execute(self, arguments):
+        wallet = PromptData.Wallet
+
+        if len(arguments) < 4:
+            print("Please specify the required parameters")
+            return None
+
+        if len(arguments) > 5:
+            # the 5th argument is the optional attributes,
+            print("Too many parameters supplied. Please check your command")
+            return None
+
+        try:
+            from_addr = wallet.ToScriptHash(arguments[0])
+        except ValueError as e:
+            print(str(e))
+            return None
+
+        asset_id = PromptUtils.get_asset_id(wallet, arguments[1])
+
+        try:
+            index = int(arguments[2])
```

Comment:
Can we also test for a negative value please. The other invalid handling is fine
```
neo> wallet split AK2nJJpJr6o664CWJKi1QRXjqeic2zRp8y NEO -1 2
No unspent assets matching the arguments.
neo> wallet split AK2nJJpJr6o664CWJKi1QRXjqeic2zRp8y NEO bla 2
Invalid unspent index value: bla
```

Reply:
Ok that's fine. Thanks for the explanation